atomic_reactor/plugins/exit_pulp_publish.py

```diff
@@ -81,8 +81,9 @@ def publish_to_crane(self, repo_prefix="redhat-"):
         self.pulp_handler.create_dockpulp()
         if not repo_prefix:
             repo_prefix = ''
-        pulp_repos = ['%s%s' % (repo_prefix, image.pulp_repo) for image in image_names]
-        self.pulp_handler.publish(pulp_repos)
+        pulp_repos = set(['%s%s' % (repo_prefix, image.pulp_repo)
```

Comment:
Sure, but isn't the pulp_repo name derived from the `name` label in Dockerfile?
`pulp_repo` does not include the registry, nor the tag.